crates/cairo-profiler/src/trace_reader/function_trace_builder.rs

```diff
@@ -15,12 +16,12 @@ use trace_data::TraceEntry;
 mod function_stack_trace;
 
 pub struct FunctionLevelProfilingInfo {
-    pub functions_stack_traces: Vec<FunctionStackTrace>,
+    pub functions_traces: Vec<FunctionCallTrace>,
     pub header_steps: Steps,
 }
 
-pub struct FunctionStackTrace {
-    pub stack_trace: Vec<FunctionName>,
+pub struct FunctionCallTrace {
```

Comment:
```suggestion
pub struct FunctionCallStackTrace {
```

Reply:
How is it better?

Reply:
because it is `CallStackTrace`. `CallTrace` Does not imply the stack structure of the data. It can be for example all called functions from the beginning.

Reply:
Got rid of this struct completely